docs/pages/admin-guides/access-controls/access-request-plugins/ssh-approval-jira.mdx

```diff
@@ -107,7 +107,7 @@ project**. Select **Kanban** for the template, then **Use template**. Click
 **Select a company-managed project**.
 
 You'll see a screen where you can enter a name for your project. In this guide,
-we assume that your project is called "Teleport Access Requests", which
+we assume that your project is called Teleport Access Requests", which
```

Comment:
I don't think we wanted to remove one of the quotes here. Remove both of them or keep it as it was originally.